site/app/controllers/admin/UsersController.php

```diff
@@ -923,9 +931,12 @@ public function uploadUserList($list_type = "classlist") {
             $exists = false;
             foreach ($existing_users as $i => $existing_user) {
                 if ($row[0] === $existing_user->getId()) {
-                    // Validate if this user has any data to update.
-                    // Did student registration section or grader group change?
-                    if ($row[4] !== $get_user_registration_or_group_function($existing_user)) {
+                    if (count($row) == 1) {
```

Comment:
Please always use `===` or `!==` for testing equality or inequality.

Reply:
Done, thanks!